src/chain.h

```diff
@@ -150,6 +153,15 @@ class CBlockIndex
     //! (memory only) Sequential id assigned to distinguish order in which blocks are received.
     uint32_t nSequenceId;
 
+    //! Index entry serial format version
```

Comment:
We need to track nSerialVersion in the CBlockIndex object.  The existing nVersion in the object is the actual block header version.  nSerialVersion is read/written to the old nVersion location at the beginning of the serialized CBlockIndex object.

The ubiquitous serialization nType/nVersion parameters have been removed in a later core release.